keystored/scripts/ssh-key-import.sh

```diff
@@ -16,7 +16,10 @@ if [ -n "$($SYSREPOCFG -d startup --export ietf-keystore)" ]; then
     exit 0
 fi
 
-if [ -f /etc/ssh/ssh_host_rsa_key ]; then
+if [ $KEYSTORED_CHECK_SSH_KEY -eq 0 ]; then
+    echo "Warning: Assuming that an external script will provide the SSH key in a PEM format at \"${KEYSTORED_KEYS_DIR}/ssh_host_rsa_key.pem\"."
+    $SYSREPOCFG -d startup -i ${STOCK_KEY_CONFIG} ietf-keystore
+elif [ -f /etc/ssh/ssh_host_rsa_key ]; then
```

Comment:
So, could you maybe change this now to `-r`?

Reply:
Added with a warning in case the file is not readable.